projects/plugins/sync-tester/composer.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I don't think we want to include this test plugin in the PR, because we don't want this merged into our codebase. It's only useful for testing the specific changes in this PR and isn't useful for testing anything else.